recipes/poster/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "poster" %}
+{% set version = "0.8.1" %}
+{% set sha256 = "af5bf45da4a916db2b638cffd9e9d6668b33020e2b8ca9f864db79b49331c6ff" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - poster
+
+about:
+  home: https://atlee.ca/software/poster/
+  license: MIT
+  license_family: MIT
+  summary: Streaming HTTP uploads and multipart/form-data encoding
+
+extra:
+  recipe-maintainers:
+    - hassanamr
```

Comment:
@synapticarbors, thanks :)
@bgruening, I will add you as soon as I get the notification for if being on feedstock. Thanks :)